dashboard/app/models/levels/blockly.rb

```diff
@@ -316,19 +316,26 @@ def blockly_level_options
     options.freeze
   end
 
-  def localized_failure_message_override
-    if should_localize? && failure_message_override
-      I18n.t("data.failure_message_overrides").
-        try(:[], "#{name}_failure_message_override".to_sym)
+  def get_localized_property(property_name)
+    if should_localize? && try(property_name)
```

Comment:
Nit: maybe clearer as `&& property_name.present?`

Reply:
Actually, is that how `present?` works? I would expect that to just check for the string value of `property_name` rather than actually inspecting the level's property

Reply:
Ah right sorry — should be `properties[property_name].present?` which I'm not sure is any cleaner.